Source/Mosa.Korlib/System/Span.cs

```diff
@@ -12,6 +12,14 @@ namespace System
 		internal readonly ByReference<T> _pointer;
 		private readonly int _length;
 
+		public T Pointer
```

Comment:
Is in the public API?

I don't see it:

https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Span.cs

Reply:
It's not, but to not make ByReference and the internal pointer public, I decided to add this property. (To make life easier, because I saw a lot of people access the internal pointer of Spans)

Reply:
It would be better to implement the `WriteUnaligned` method in the `Unsafe` class and create the `MemoryMarshal` class and add the few relevant functions (like `GetReference`).

Our ultimate goal in the future is to use the pre built corelib from the official runtime so we want to stick as close to the public API as possible.

Here's the links to the classes:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/Internal/Runtime/CompilerServices/Unsafe.cs
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/MemoryMarshal.cs

Reply:
Implemented.